<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginScreen</class>
 <widget class="QWidget" name="LoginScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>900</width>
       <height>600</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>900</width>
       <height>600</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
     <widget class="QLabel" name="titleLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>900</width>
        <height>100</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>900</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>900</width>
        <height>100</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Enter Players name</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>900</width>
        <height>500</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>900</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>900</width>
        <height>500</height>
       </size>
      </property>
      <widget class="QLineEdit" name="lineEditP2">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>150</y>
         <width>350</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
       <property name="placeholderText">
        <string>Player name...</string>
       </property>
      </widget>
      <widget class="QPushButton" name="continueBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>375</x>
         <y>350</y>
         <width>150</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Continue</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEditP1">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>80</y>
         <width>350</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="maxLength">
        <number>20</number>
       </property>
       <property name="placeholderText">
        <string>Player name...</string>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>80</y>
         <width>110</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>player 1:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>150</y>
         <width>110</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <underline>false</underline>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
</string>
       </property>
       <property name="text">
        <string>player 2:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="errorLabel">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>240</y>
         <width>471</width>
         <height>71</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(213, 0, 0);</string>
       </property>
       <property name="text">
        <string>Erros here...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditP1</tabstop>
  <tabstop>lineEditP2</tabstop>
  <tabstop>continueBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
